components/raftstore/src/store/fsm/store.rs

```diff
@@ -782,14 +693,13 @@ impl<EK: KvEngine, ER: RaftEngine, T: Transport> PollHandler<PeerFsm<EK, ER>, St
 {
     fn begin(&mut self, _batch_size: usize) {
         self.previous_metrics = self.poll_ctx.raft_metrics.clone();
-        self.poll_ctx.processed_fsm_count = 0;
         self.poll_ctx.pending_count = 0;
+        self.poll_ctx.ready_count = 0;
         self.poll_ctx.sync_log = false;
         self.poll_ctx.has_ready = false;
         self.poll_ctx.disk_usage = disk::get_disk_status(self.poll_ctx.store.get_id());
-        self.timer = TiInstant::now_coarse();
+        self.timer = TiInstant::now();
```

Comment:
Is it expected to change it?

Reply:
Yes. The store loop is faster than before and usually less than 1ms.